docs/.vuepress/config.js

```diff
@@ -161,7 +161,7 @@ module.exports = {
             ['/guide/advanced-usage', 'Advanced usage'],
             ['/guide/configuration-options', 'Configuration options'],
             ['/guide/license-key', 'License key'],
-            ['/guide/known-limitations', 'Known limitations'],
+            ['/guide/compatibility', 'Compatibility'],
```

Comment:
I called this page simply `Compatibility`. Do you think it should be more descriptive e.g. `Compatibility with other software`?

Reply:
@sequba – I'm wondering if "compatibility" is the right term here: it usually means that software works **together** with other software, and implies some sort of integration. I think what we're talking about here is "similarity" to other spreadsheet software and "conformity" to the OpenDocument standard.

In fact, what we're really focusing on is **differences** between HF and other spreadsheet software, so I'd title the page "Differences with other spreadsheets", like [here](https://www.ibm.com/docs/en/i/7.3?topic=qshell-differences-other-interpreters) or [here](https://clojure.org/reference/lisps).

I'll prepare a PR with my suggestions :)

Reply:
Well, I think the most important for our users is:
- will the formulas exported from Microsoft Excel/Google Sheets work when imported into HyperFormula?
- will the results of computations be the same as in other software? if not, what differences could be expected?
- how to configure HyperFormula to work as similar to Microsoft Excel/Google Sheets as possible?

The aim of this page is to answer these questions and its title should suggest that it will. I trust you'll find the best term for that ;)

Reply:
@sequba - nice, I didn't know that! Indeed, this puts the "Compatibility" term in a different light. WIP: #1054.